README.markdown

```diff
@@ -6669,6 +6673,78 @@ This feature was first introduced in the `v0.7.3` release.
 
 [Back to TOC](#nginx-api-for-lua)
 
+ngx.shared.DICT.capacity
+---------------------
+**syntax:** *capacity_bytes = ngx.shared.DICT:capacity()*
+
+**context:** *init_by_lua&#42;, set_by_lua&#42;, rewrite_by_lua&#42;, access_by_lua&#42;, content_by_lua&#42;, header_filter_by_lua&#42;, body_filter_by_lua&#42;, log_by_lua&#42;, ngx.timer.&#42;, balancer_by_lua&#42;, ssl_certificate_by_lua&#42;, ssl_session_fetch_by_lua&#42;, ssl_session_store_by_lua&#42;*
+
+**requires:** `resty.core.shdict` or `resty.core`
+
+Retrieving the capacity in bytes for the shm-based dictionary [ngx.shared.DICT](#ngxshareddict) declared with
+the [lua_shared_dict](#lua_shared_dict) directive.
+
+
+Example:
+
+```lua
+
```

Comment:
Extra blank line here?

Please fix other similar places.

Reply:
I did put this extra blank line here similar to other places like `ngx.shaed.DICT.ttl`
https://raw.githubusercontent.com/openresty/lua-nginx-module/97fbeb0bef1aa85e758210d58063376de8eaed31/README.markdown

Reply:
This line is inserted by the `wiki2markdown.pl` script. Until recently (and probably still the case), the README is not to be edited directly. See the note at the beginning:
```
<!---
Don't edit this file manually! Instead you should generate it by using:
    wiki2markdown.pl doc/HttpLuaModule.wiki
-->
```

Reply:
@thibaultcha I see. Never mind then.

Reply:
@thibaultcha Thanks for your comment. I didn't notice that comment. I'll update doc/HttpLuaModule.wiki and regenerate README.markdown.

Reply:
I've updated doc/HttpLuaModule.wiki, regenerated README.markdown and confirmed there is no difference between the generated version and the README.markdown in the previous commit.